src/vcpkg/build.cpp

```diff
@@ -973,8 +973,28 @@ namespace vcpkg
         }
 
         const BuildInfo build_info = read_build_info(fs, paths.build_info_file_path(action.spec));
-        const size_t error_count =
-            perform_post_build_lint_checks(action.spec, paths, pre_build_info, build_info, scfl.source_location);
+        const size_t error_count = [&]() {
+            struct FileSink : MessageSink
+            {
+                WriteFilePointer out_file;
+                StringView stdoutlog;
```

Comment:
Member `StringView` is a bit scary because it makes the lifetime here depend on outliving the input. And worse, that's only "discoverable" when an error happens.

```suggestion
                Path stdoutlog;
```